src/intents.cs

```diff
@@ -7818,10 +7818,16 @@ interface INSearchForAccountsIntentResponse {
 	[BaseType (typeof (INIntent))]
 	interface INSearchForNotebookItemsIntent {
 
+		[Deprecated (PlatformName.WatchOS, 4, 2, message: "Use the 'notebookItemIdentifier' .ctor instead.")]
+		[Deprecated (PlatformName.iOS, 11, 2, message: "Use the 'notebookItemIdentifier' .ctor instead.")]
```

Comment:
Let's not use `.ctor` we should assume not everyone knows what it means.

Also maybe we could be more specific on the parameter so something like: `Use the constructor with `string notebookItemIdentifier` instead.`